<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>welcomePage</class>
 <widget class="QDialog" name="welcomePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>welcomePage</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(../untitled.jpg)</string>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>521</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>29</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Formula1 Tournament Management System</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="addNewTournamentPushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>500</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;</string>
   </property>
   <property name="text">
    <string>Add New Tournament</string>
   </property>
  </widget>
  <widget class="QPushButton" name="adminPanelPushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>500</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;</string>
   </property>
   <property name="text">
    <string>Admin panel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="statisticsPushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>500</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;</string>
   </property>
   <property name="text">
    <string>Statistics</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
